test/test_images.jl

```diff
@@ -4,12 +4,14 @@ using DataFrames
 import SloanDigitalSkySurvey: SDSS, WCSUtils, PSF
 
 using Celeste: Types, SampleData
-import Celeste: ModelInit, SkyImages, ElboDeriv
+import Celeste: ModelInit, SkyImages, ElboDeriv, Synthetic
 
 
 println("Running SkyImages tests.")
 
-field_dir = joinpath(dat_dir, "sample_field")
+const datadir = joinpath(Pkg.dir("Celeste"), "test", "data")
+
+field_dir = joinpath(datadir, "sample_field")
```

Comment:
>
> It's weird we have both dat_dir defined in SampleData.jl and datadir
> defined runtests.jl. SampleData.jl probably shouldn't know about the test
> data's location.
>

I agree. Probably SampleData.jl should eventually be moved to tests, as its
not necessary for the core functionality in the app. This is something to
think about after SC though!